### PR TITLE
Add recipe for sos-julia (regenerated from #9366)

### PR DESCRIPTION
**Pull request by BoPeng**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by BoPeng** [comment]
@conda-forge/help-julia This is ready to be reviewed.

File: recipes/sos-julia/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - export JUPYTER_DATA_DIR=${PREFIX}/share/jupyter
+    - export JUPYTER=$(which jupyter)
+    - julia -e "using Pkg;Pkg.add([\"IJulia\", \"Feather\", \"DataFrames\", \"NamedArrays\"])"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```
Comment (scopatz) [inline recipes/sos-julia/meta.yaml]: Python should probably be a run requirement too

Comment (BoPeng) [reply recipes/sos-julia/meta.yaml]: done.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**